packages/utilities/psammead-locales/moment/helpers/jalaali.js

```diff
@@ -0,0 +1,30 @@
+const moment = require('moment-timezone/moment-timezone');
+const jalali = require('jalaali-js');
+
+function getJalaliString(gregorianMoment, jalaliMonths) {
+  const jalaliDate = jalali.toJalaali(
+    gregorianMoment.year(),
+    gregorianMoment.month() + 1,
+    gregorianMoment.date(),
+  );
+  const output = `${jalaliDate.jd} ${jalaliMonths[jalaliDate.jm - 1]} ${
+    jalaliDate.jy
+  }`;
+  return output;
+}
+
+function addJalaliDate(locale, jalaliMonths, jalaliFormats, gregorianString) {
```

Comment:
This function will not be needed in this format.
The reason is, the markup for the two dates on a page should be this, with the gregorian calendar in a time element with the datetime that search engines use to get the date value. Then a dash with the Jalaali date in another time element. 
```
<time datetime="2019-01-31">۱ ژانویه ۲۰۱۹</time> - <time>۱۱ بهمن ۱۳۹۷</time>
```

Reply:
To clarify, we only want these utilities to export the dates as strings, not as components with `<time>` tags. It will be within the `TimestampContainer` package where we use the `<time>` element. 


We'll want the API to be something that is simple to use, e.g.:

`jalaaliPersianDatetime(unixTimestamp)`
`jalaaliPashtoDatetime(unixTimestamp)`
And these exports can abstract away the usage of the `stringHelper.useEasternNumerals`, `getJalaaliDatetime` and `jalaliMonths`

Reply:
Agree we don't want to include HTML elements here, though the API can't be exactly as suggested here as the implementation (as in `moment-ws`) relies on Moment.js' `postformat` hook to append the extra information after the timestamp has been formatted.

Reply:
Discussed this with @sareh and we decided it's OK to return a single human-readable string to be the content of a a single `time` element. Therefore we do not require an API to return both timestamps separately.